packages/mask/src/plugins/Collectible/SNSAdaptor/ArticleTab.tsx

```diff
@@ -41,7 +43,7 @@ export function ArticleTab(props: ArticleTabProps) {
     const { asset } = CollectibleState.useContainer()
 
     if (!asset.value) return null
-    const resourceUrl = asset.value.image_url || asset.value.animation_url
+    const resourceUrl = renderMedia(asset.value.image_url || asset.value.animation_url)
```

Comment:
You need to inject it by inserting an iframe element. Also, a fallback plan is required if the iframe posts an error message out of it.